### DISCUSSION TITLE
Uncaught (in promise) Cannot perform range filter - invalid type. error at application after updating to 2.13.17

### DISCUSSION THREAD

**Discussion by gmentzik** [Budibase/budibase#12454] [Help] [answered]
Hi there,
After updating self host community edition from 2.8.10 to 2.13.17, i am seeing the following error at at least two of my application screens that are relying heavily on custom Mysql queries and table and pagination is also created by the custom query and JS.
**Uncaught (in promise) Cannot perform range filter - invalid type**
These pages used to work fine on 2.8.10 but now are unresponsive as waiting icon at related components keeps showing forever. I am attaching the console log export from browser.
Errors occurs at preview and at published app if I proceed publishing it in 2.8.10 or at 2.13.17

[budibase_error-1701167236758.log](https://github.com/Budibase/budibase/files/13487045/budibase_error-1701167236758.log)
[budibase_error-1701167236758-client1.log](https://github.com/Budibase/budibase/files/13487046/budibase_error-1701167236758-client1.log)

**Comment by manwithpowers** [comment]
There's a report for this, looks like it has to do with JS/conditionals that reference form fields that contain integers: https://github.com/Budibase/budibase/issues/12448
If you roll back to 2.13.13 you should be alright.

**Reply by gmentzik** [reply, marked as answer]
I can revert only to 2.8.10 as all versions I have tried since then have various issues. Latest major issue was breaking the DB Relationships functionalities. 2.8.10 is the latest for me that everything worked fine for my app.

BTW the fix suggested at link https://github.com/Budibase/budibase/issues/12448 , (use binding instead of numbers at conditionals) at custom pagination buttons allows the page to load correctly! Thank you @manwithpowers
